### PR TITLE
A How-To Guide for Setting Up OTBR with the Nordic nRF52840 Dongle

### PR DESCRIPTION
**Pull request by MonicaisHer**
This PR is a draft for wiki page, so it will not be merged into the main branch.

### CODE REVIEW

File: wiki.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+[23931.133681] cdc_acm 1-8:1.0: ttyACM0: USB ACM device
+```
+
+### References:
+- https://github.com/openthread/openthread/blob/main/src/posix/README.md#nordic-semiconductor-nrf52840
+- https://developer.nordicsemi.com/nRF_Connect_SDK/doc/1.9.99-dev1/matter/openthread_rcp_nrf_dongle.html
+- https://github.com/canonical/openthread-border-router-snap/wiki/Commission-and-control-a-Matter-Thread-device-via-the-OTBR-Snap
+- https://github.com/openthread/ot-nrf528xx/blob/main/src/nrf52840/README.md#mass-storage-device-known-issue
+- https://github.com/crownstone/bluenet/issues/81#issuecomment-561257090
+- https://github.com/makerdiary/nrf52840-mdk-usb-dongle/issues/56#issuecomment-1322301257
```
Comment (farshidtz) [inline wiki.md:49]: Please add link titles. Otherwise, these aren't so useful.

File: wiki.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+This how-to guide will help you set up and configure OTBR with Nordic nRF52840 Dongle. 
+For additional information and troubleshooting, please refer to the provided references.
```
Comment (farshidtz) [inline wiki.md:52]: Is this meant to be a part of intro?

File: wiki.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# A How-To Guide for Setting Up OTBR with the Nordic nRF52840 Dongle
```
Comment (farshidtz) [inline wiki.md:1]: This isn't a how-to by itself as it mostly just links to other pages. 
```suggestion
# Setup OpenThread Border Router with nRF52840 Dongle
```

File: wiki.md (hunk 2 above)
Comment (MonicaisHer) [reply wiki.md]: It has been moved to the beginning now

File: wiki.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+Following the instructions [here](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/1.9.99-dev1/matter/openthread_rcp_nrf_dongle.html):
+
+#### Debugging tips 
+
+##### Finding the device name
+
+During the setup process, ensure that the dongle device is connected and recognized as expected at `/dev/ttyACM0`. 
+The `dmesg` command can help with this verification. 
+Here's an example of how to use it:
```
Comment (farshidtz) [inline wiki.md:19]:
```suggestion
For flashing the firmware and using the RCP, you need to find the device name assigned to the dongle.

The simplest way to do so is to run `dmesg` with the `-w`/`--follow` flag before connecting the device. 
Here is an example, showing device name `ttyACM0` which means that the device path on the system would be `/dev/ttyACM0`:
```

**Comment by farshidtz** [comment]
Looks ready to move over.

File: wiki.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+If you encounter an error related to pip during the installation of nRF Util, update pip with the following commands:
+```
+python3 -m pip install -U pip
+python3 -m pip install -U nrfutil
+```
+
+### Configure and run OTBR snap:
+
+Now that the RCP firmware has successfully flashed, it's time to configure and run the OTBR Snap. 
+Follow these steps in this [guide](https://github.com/canonical/openthread-border-router-snap/wiki/Commission-and-control-a-Matter-Thread-device-via-the-OTBR-Snap#install-and-configure-the-otbr-snap)
```
[the remark this answers is not in the document]
Comment (MonicaisHer) [reply wiki.md:79]: It  has been updated to relative URL in wiki now: https://github.com/canonical/openthread-border-router-snap/wiki/Setup-OpenThread-Border-Router-with-nRF52840-Dongle

**Pull request closed without merging**